### DISCUSSION TITLE
Trying to install this extension in GitHub workflow requires GH_TOKEN env variable

### DISCUSSION THREAD

**Discussion by AdamRabcan** [Link-/gh-token#38] [Q&A] [answered]
I am trying to use this extension so that I can generate GitHub token from APP_ID and Private Key but when I want to install required plug in I get an error saying the GH_TOKEN is missing. The issue is I do not have this token because I need this extension to create it right? If I use just a random placeholder value for this env variable I get Bad credentials error... How can I download and install this extension without being authenticated pls?

**Comment by AdamRabcan** [comment]
![image](https://github.com/Link-/gh-token/assets/109140115/c8d4f06d-938c-42ae-a3d5-31a18b35784e)

**Reply by AdamRabcan** [reply]
![image](https://github.com/Link-/gh-token/assets/109140115/9d0a77f0-ccbd-4380-b15d-dd4ed001669a)

**Reply by Link-** [reply, marked as answer]
Do not use this extension in your workflows, instead use this action: https://github.com/actions/create-github-app-token

This extension ins designed for on your machine use. While it can be used in workflows, there are better alternatives like the one I mentioned.

**Reply by AdamRabcan** [reply]
Thank you ! I am going to check that out.
